app/grandchallenge/hanging_protocols/forms.py

```diff
@@ -116,3 +129,91 @@ def _validate_slice_plane_indicator(self, *, viewport, viewport_names):
                 error=f"Viewport {viewport['viewport_name']} has a slice_plane_indicator that is the same as the viewport_name.",
                 field="json",
             )
+
+
+class ViewContentExampleMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance:
+            interface_slugs = self.instance.interfaces.values_list(
+                "slug", flat=True
+            )
+
+            if interface_slugs.count() > 0:
+                self.fields[
+                    "view_content"
+                ].help_text += f"The following interfaces are used in your {self.instance._meta.verbose_name}: {oxford_comma(interface_slugs)}. "
+
+            view_content_example = self.generate_view_content_example()
+
+            if view_content_example:
+                self.fields[
+                    "view_content"
+                ].help_text += f"Example usage: {view_content_example}. "
+            else:
+                self.fields[
+                    "view_content"
+                ].help_text += "No interfaces of type image, chart, pdf, mp4, thumbnail_jpg or thumbnail_png are used. At least one interface of those types is needed to configure the viewer. "
+
+        self.fields["view_content"].help_text += format_lazy(
+            'Refer to the <a href="{}">documentation</a> for more information',
+            reverse("documentation:detail", args=["viewer-content"]),
+        )
+
+    def generate_view_content_example(self):
+        images = [
+            interface.slug
+            for interface in self.instance.interfaces.all()

```

Comment:
The `.all()` should not be necessary here, right?

Reply:
Done